libkineto/src/init.cpp

```diff
@@ -143,10 +144,19 @@ void libkineto_init(bool cpuOnly, bool logOnError) {
 #endif
 
 #ifdef HAS_CUPTI
+  bool initCupti = true;
+
+  const char* kDisableCuptiEnvVar = "KINETO_DISABLE_CUPTI";
+  const char* disableCuptiEnvVal = getenv(kDisableCuptiEnvVar);
+  if (disableCuptiEnvVal) {
+    initCupti = atoi(disableCuptiEnvVal) ? false : true;
+    LOG(INFO) << "Setting initCupti = " << initCupti << " from environment " << kDisableCuptiEnvVar << "=" << disableCuptiEnvVal;
```

Comment:
Don't we have any function to handle ENV variables?
Could you use/add it as an util?